packages/synapse-react-client/src/components/AccessRequirementRelatedProjectsList/AccessRequirementRelatedProjectsList.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { AccessRequirementSearchRequest } from '@sage-bionetworks/synapse-types'
+import { useSearchAccessRequirementsInfinite } from '../../synapse-queries'
+import { EntityLink } from '../EntityLink'
+import { Alert, List, ListItem, Typography } from '@mui/material'
+
+export type AccessRequirementRelatedProjectsListProps = {
+  accessRequirementId: number
+}
+
+export const AccessRequirementRelatedProjectsList = (
+  props: AccessRequirementRelatedProjectsListProps,
+) => {
+  const searchRequest: Omit<AccessRequirementSearchRequest, 'nextPageToken'> = {
+    ids: [props.accessRequirementId],
+  }
+
+  const { data, isError, error } =
+    useSearchAccessRequirementsInfinite(searchRequest)
+  const ar = data?.pages.flatMap(page => page.results)[0] ?? undefined
+
+  const nRelatedProjects = ar?.relatedProjectIds.length || 0
+  const relatedProjectsTitle = `${nRelatedProjects} project${
+    nRelatedProjects === 1 ? '' : 's'
+  }${nRelatedProjects > 0 ? ':' : ''}`
```

Comment:
We already have `pluralize` as a dependency, so you can just use that.

Reply:
Oh great, thanks! Fixed in the next commit